services/src/pro/contexts/migrations/current_schema.sql

```diff
@@ -216,3 +216,12 @@ CREATE TABLE ml_models (
     id uuid PRIMARY KEY,
     content text NOT NULL
 );
+
+CREATE TABLE oidc_session_tokens (
+    session_id uuid REFERENCES sessions (id) ON DELETE CASCADE NOT NULL,
```

Comment:
add unique constraint? or can there be multiple tokens per session?

Reply:
Added a Primary Key, or would Unique be preferable for some reason? c09d87f913f283a000cff7c3c61c9aefb288fd1c